src/commands/environments/create.js

```diff
@@ -28,7 +29,8 @@ class CreateCommand extends Command {
           return;
         }
       }
-      this.error(error, { exit: 1 });
+      catchUnexpectedError(error);
```

Comment:
`this.error()` is differrent than `console.error()` + `this.exit()` ! Please use `this.error()`.
Furthermore, catchUnexpectedError is qlways followed by exit(1). I think you should include the exit inside the catchUnexpectedError

Reply:
I left a comment, see below.